crmd/remote_lrmd_ra.c

```diff
@@ -80,6 +80,10 @@ typedef struct remote_ra_data_s {
     enum remote_migration_status migrate_status;
 
     gboolean active;
+    gboolean is_maintenance; /* kind of complex to determine from crmd-context
+                              * so we have it signalled back with the
+                              * transition from pengine
+                              */
```

Comment:
Right - that is still missing. And I'm having unexpected hard time with it as pseudo-actions are
handled directly inside tengine instead of being handed over to crmd (local or routed).